data/multiplayer/eras.lua

```diff
@@ -73,26 +68,36 @@ res.turns_over_advantage = function()
 				end
 				-- Up to here
 				local total = units + team.gold + income
-				side_comparison = side_comparison .. string.format( tostring( _ "<span foreground='#%02x%02x%02x'>Side %d</span>:  Income score = %d  Unit score = %d  Gold = %d") .. "\n" .. tostring( _ "Grand total: <b>%d</b>") .. "\n",
-				r, g, b, side, income, units, team.gold, total)
+				-- po: In the end-of-match summary, any side that still has units left
+				local side_text = _ "<span foreground='$side_color'>Side $side_number</span>:  Income score = $income  Unit score = $units  Gold = $gold\nGrand total: <b>$total</b>"
+				side_comparison = side_comparison .. side_text:vformat{side_color = side_color, side_number = side, income = income, units = units, gold = team.gold, total = total} .. "\n"
 				if total > total_score then
-					color = string.format("#%02x%02x%02x", r, g, b)
+					winners_color = side_color
 					winning_sides = {side}
-					tie = false
 					total_score = total
 				elseif total == total_score then
 					table.insert(winning_sides, side)
-					tie = true
 				end
 			end
 		end
 	end
 
-	if tie then
-		local last_winning_side = table.remove(winning_sides)
-		side_comparison = side_comparison .. string.format( "\n" .. tostring( _ "Sides %s and %d are tied."), table.concat(winning_sides, ", "), last_winning_side)
+	if #winning_sides == 0 then
+		-- every side either has no units or has a negative score
+	elseif #winning_sides == 1 then
+		-- po: In the end-of-match summary, there's a single side that's won.
+		local comparison_text = _ "<span foreground='$side_color'>Side $side_number</span> has the advantage."
+		side_comparison = side_comparison .. "\n" .. comparison_text:vformat{side_number = winning_sides[1], side_color = winners_color}
+	elseif #winning_sides == 2 then
+		-- po: In the end-of-match summary, there's a two-way tie (this is only used for exactly two winning teams)
+		-- Separated from the three-or-more text in case a language differentiates "two sides" vs "three sides".
+		local comparison_text = _ "Sides $side_number and $other_side_number are tied."
+		side_comparison = side_comparison .. "\n" .. comparison_text:vformat{side_number = winning_sides[1], other_side_number = winning_sides[2]}
 	else
-		side_comparison = side_comparison .. string.format( "\n" .. tostring( _ "<span foreground='%s'>Side %d</span> has the advantage."), color, winning_sides[1])
+		local winners = stringx.format_conjunct_list("", winning_sides)
+		-- po: In the end-of-match summary, three or more teams have all tied for the best score. $winners contains the result of formatting the conjunct list.
+		local comparison_text = _ "Sides $winners are tied."
+		side_comparison = side_comparison .. "\n" .. comparison_text:vformat{winners = winners}
 	end
 	wesnoth.fire("message", { message = side_comparison, speaker = "narrator", image = "wesnoth-icon.png"})
```

Comment:
Maybe this is out-of-scope for this PR, but I _really_ think this should use `gui.show_popup` instead of the current `gui.show_narration`.

Reply:
Once the CI is running again I'll push an update with this.